### PR TITLE
Adding About Me page

### PR DESCRIPTION
**Pull request by tomat-suppe**
* About Me page has been added
* - Page lists logged in user's Cheeps and who they follow (with hyperlinks)
* Forget Me has been moved here

Co-authored-by: Sophie Gabriela <slak@itu.dk>

### CODE REVIEW

**Review by JaSNOHo — APPROVED** [review]
latm

**Review by Thyme-Cat — APPROVED** [review]
l-pretty-tm

**Review by llsa-itu — APPROVED** [review]
lgtm

File: src/ChirpWeb/wwwroot/css/style.css [hunk 1]
```diff
@@ -272,6 +272,14 @@ FollowUnfollowButton {
 	cursor: pointer;
 }
 
+forgetButton{
+	display: initial;
+	background-color: rgb(252, 253, 253);
+	color: #1717e1;
+	text-decoration: none;
+	cursor: pointer;
```
[the remark this answers is not in the document]
Comment (tomat-suppe) [reply src/ChirpWeb/wwwroot/css/style.css]: deleted <3

**Review by HenrijetteOest — APPROVED** [review]
l very gtm :))

**Pull request merged**